fire/completion.py

```diff
@@ -308,7 +308,7 @@ def MemberVisible(component, name, member, class_attrs=None, verbose=False):
     return False
   if verbose:
     return True
-  if member in (absolute_import, division, print_function):
+  if type(member) in (absolute_import, division, print_function):
```

Comment:
I don't think this is quite the check we want.

In Python 3 type(absolute_import) is __future__._Feature, which is fine.
However in Python 2 type(absolute_import) is instance which is too broad.

Instead we can do `if (member is absolute_import or member is division or member is print_function)`

Reply:
Sounds good @dbieber I'll make the change and push again